src/components/co-staking/CoStakingCalculator.jsx

```diff
@@ -0,0 +1,200 @@
+        </label>
+        <input
+          type="number"
+          id="total-co-stakers-weight"
+          step="0.01"
+          min="0"
+          placeholder="Leave empty for ratio calculation only"
+          value={totalNetworkWeight}
+          onChange={(e) => setTotalNetworkWeight(e.target.value)}
+          className={styles.input}
+        />
+        <small className={styles.helperText}>
+          If left empty, calculator will show your weight and ratio only
+        </small>
+      </div>
+
+      <button onClick={calculateRewards} className={styles.calculateButton}>
+        Calculate Rewards
+      </button>
+
+      {results && (
+        <div className={styles.results}>
+          <h4>Results:</h4>
+          <div className={styles.resultsContent}>
+            {/* Basic Info */}
+            <div className={styles.infoBox}>
+              <strong>Your Co-Staking Weight:</strong> {results.userWeight} BTC-equivalent<br />
+              <strong>Eligible BTC:</strong> {results.eligibleBTC} / {results.maxEligibleBTC} BTC<br />
+              <strong>BABY per BTC Ratio:</strong> {results.ratio} BABY/BTC<br />
+              <strong>Efficiency:</strong> {results.efficiency}% (100% = optimal ratio of 20,000 BABY per BTC)
+              {parseFloat(results.userWeight) > 0 && (
+                <div style={{ marginTop: '10px', padding: '8px', background: '#d4edda', borderRadius: '4px', fontSize: '14px' }}>
+                  ✅ <strong>You have co-staking weight!</strong> Even small amounts earn rewards proportionally.
+                </div>
+              )}
+            </div>
+
+            {/* Recommendations */}
+            {results.needsMoreBaby && (
+              <div className={`${styles.recommendationBox} ${styles.warningBox}`}>
+                <strong>💡 Recommendation:</strong><br />
+                To maximize rewards, stake {results.needsMoreBaby.toLocaleString()} more BABY tokens<br />
+                (Target: {(parseFloat(btcAmount) * CO_STAKING_FACTOR).toLocaleString()} BABY for {btcAmount} BTC)
+              </div>
+            )}
+
+            {results.hasExcessBaby && results.optimalBTC && (
+              <div className={`${styles.recommendationBox} ${styles.infoBox}`}>
+                <strong>💡 Note:</strong><br />
+                You have excess BABY. Consider staking {results.optimalBTC.toFixed(4)} BTC to maximize efficiency<br />
+                (Current: {btcAmount} BTC, Optimal: {results.optimalBTC.toFixed(4)} BTC)
+              </div>
+            )}
+
+            {/* Reward Calculations */}
+            {results.rewards ? (
+              <>
+                <div className={`${styles.recommendationBox} ${styles.successBox}`}>
+                  <h4>Estimated Rewards (Annual):</h4>
+                  <div className={styles.rewardAmount}>
+                    {results.rewards.annual.toLocaleString(undefined, { maximumFractionDigits: 2 })} BABY/year
```

Comment:
The results are in green and hard to read in a "successBox" css styling. Can we update this to just be White? 
<img width="658" height="704" alt="image" src="https://github.com/user-attachments/assets/7c91ee0b-7e96-44cd-ae06-398d2120d526" />